### PR TITLE
Times of India news scraper

### PR DESCRIPTION
**Pull request by rutujadhanawade**
- This is python script for scraping the TOI
- #92

### CODE REVIEW

**Comment by rutujadhanawade** [comment]
@Neilblaze @ankitdobhal please review

**Comment by rutujadhanawade** [comment]
@Neilblaze please review this PR.

File: Web-Scraping/Times_of_india/Times_of_india.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+response = requests.get(url)
+# Make the html soup object
+soup = BeautifulSoup(response.content, 'html.parser')
+
+print("\t!!!**The Times of India**!!!")
+today = datetime.date.today()
+print(today.strftime('\tThe date %d, %b %Y'))
+
+# scrping times of India in four domains:
+print("\n\t\t****Flash news****")
```
Comment (Neilblaze) [inline Web-Scraping/Times_of_india/Times_of_india.py]: edit this to `"\n\t\t**** Flash news ****"`

**Review by Neilblaze — CHANGES_REQUESTED** [review]

File: Web-Scraping/Times_of_india/Times_of_india.py (hunk 1 above)
Comment (rutujadhanawade) [reply Web-Scraping/Times_of_india/Times_of_india.py]: Done @Neilblaze

**Review by Neilblaze — APPROVED** [review]
LGTM :+1:

**Comment by rutujadhanawade** [comment]
@ankitdobhal please review and merge it.

**Review by powerexploit — APPROVED** [review]

**Pull request merged**